src/script/luna/mememu.cpp

```diff
@@ -487,7 +487,7 @@ class SMBXMemoryEmulator
         insert(0x00B2C882, &MenuMode); // Current menu mode
 
         // insert(0x00B2C884, ???}; // Key Released!!!
-        insert(0x00B2C894, &BlocksSorted);
+        // insert(0x00B2C894, &BlocksSorted); // removed by block quadtree
```

Comment:
Better to give the fake field that will be always "true", just to not confuse the "ABI"

Reply:
Okay, I fixed it by myself :fox_face: